backend/app/api/api_v1/routers/search.py

```diff
@@ -21,23 +32,44 @@
 _OPENSEARCH_INDEX_CONFIG = OpenSearchQueryConfig()
 
 
-@search_router.post(
-    "/searches",
-    response_model=SearchResponseBody,
-)
+@search_router.post("/searches", response_model=SearchResponseBody)
 def search_documents(
     request: Request,
     search_body: SearchRequestBody,
     current_user=Depends(get_current_active_db_user),
 ):
-    """Search for documents matching the search criteria."""
+    """Search for documents matching the search criteria.
+
+    TODO: Don't return documents with no source urls?
+    """
     logger.info(
         "Search request",
         extra={"props": {"search_request": json.loads(search_body.json())}},
     )
 
-    return _OPENSEARCH_CONNECTION.query(
-        search_request_body=search_body,
-        opensearch_internal_config=_OPENSEARCH_INDEX_CONFIG,
-        preference=str(current_user.id),
+    if search_body.query_string:
+        """When a query string is given - hand off the complete search to OpenSearch"""
+        return _OPENSEARCH_CONNECTION.query(
+            search_request_body=search_body,
+            opensearch_internal_config=_OPENSEARCH_INDEX_CONFIG,
+            preference=str(current_user.id),
+        )
+    else:
+        """When no query string - search using RDS"""
+        return browse_rds(search_body)
+
+
+@search_router.post(
+    "/searches/country/{geography_id}",
+    summary="Gets documents associated with a country by geoography id.",
+    response_model=SearchByCountryResponse,
+)
+def search_by_country(
```

Comment:
I've looked at this a couple of times now & keep wondering if this is really a "search"... even the summary string describes it as "get me a summary of this country", rather than "search for country documents".

Would it hurt to update this URL to be something more like `GET /country_summaries/{geography_id}`?